### PR TITLE
SOLVED A MISSING MARGIN ON SUBSCRIBER DETAIL VIEW

### PR DESCRIPTION
**Pull request by elabcius5**
This PR solves a deferct described in this card:

https://trello.com/c/aRtdxHyV/14

pr8

### CODE REVIEW

**Comment by daniellemariani** [comment]
LG

File: app/src/main/res/layout/fragment_subscribers_detail.xml [hunk 1]
```diff
@@ -144,6 +144,7 @@
         android:id="@+id/list_repositories"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
+        android:layout_marginBottom="@dimen/margin_botton_recycler_view"
```
Comment (lucaslabs) [inline app/src/main/res/layout/fragment_subscribers_detail.xml]: Please set just `margin` so the 4 sides (top, left, right and bottom) will have a margin.

File: app/src/main/res/values/dimens.xml [hunk 2]
```diff
@@ -18,4 +18,5 @@
     <dimen name="margin_left_counters_subscriber_profile">5dp</dimen>
     <dimen name="txt_size_subsciber_detail">10sp</dimen>
     <dimen name="splash_title_size">25sp</dimen>
+    <dimen name="margin_botton_recycler_view">10dip</dimen>
```
Comment (lucaslabs) [inline app/src/main/res/values/dimens.xml]: A `5dp` margin will be the best fit. You can create a `margin_common = 5dp` entry.

File: app/src/main/res/layout/fragment_subscribers_detail.xml [hunk 3]
```diff
@@ -144,7 +144,10 @@
         android:id="@+id/list_repositories"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
-        android:layout_marginBottom="@dimen/margin_botton_recycler_view"
+        android:layout_marginBottom="@dimen/margin_common"
```
Comment (lucaslabs) [inline app/src/main/res/layout/fragment_subscribers_detail.xml]: there is a `margin` property. Just use it.

**Comment by lucaslabs** [comment]
LG

**Pull request merged**